GliaWidgets/Component/Bubble/BubbleView.swift

```diff
@@ -44,6 +54,27 @@ class BubbleView: UIView {
         ).cgPath
     }
 
+    func showOnHoldView() {
+        guard
```

Comment:
Since `show/hide` methods are not synced and we don't have tests for it, I'd like to propose to call `onHoldView?.removeFromSuperview()` on the beginning of this method, to be sure that after accidentally called `showOnHoldView` we will not have an additional `onHoldView` subview.

Reply:
Added